lib/model/query/forms.js

```diff
@@ -168,11 +168,72 @@ update.audit = (form, data) => (log) => log('form.update', form, { data });
 
 const _updateDef = (form, data) => ({ one }) => one(updater(form.def, data));
 
-const del = (form) => ({ run, Assignments }) =>
-  Promise.all([ run(markDeleted(form)), Assignments.revokeByActeeId(form.acteeId) ]);
+const del = (form) => ({ run, Assignments, Forms }) =>
+  Promise.all([ run(markDeleted(form)), Assignments.revokeByActeeId(form.acteeId) ])
```

Comment:
consider just purging and not having to check it and mark deleted first

Reply:
might need to revoke by actee id still even if form purged without being deleted first